app/modules/transcribe/toolbar.directive.js

```diff
@@ -15,15 +15,18 @@ function transcribeToolbar($timeout, localStorageService, SubjectsFactory, Tools
     return directive;
 
     function transcribeToolbarLink(scope) {
-        var vm = scope.vm;
+        var current = SubjectsFactory.current.data.id,
+            vm = scope.vm;
         vm.tools = ToolsFactory;
-        if (SubjectsFactory.current) {
+        scope.$watch(current, function () {
             vm.metadata = SubjectsFactory.current.data.metadata;
```

Comment:
Also, it probably IS changing, you just need to run a scope digest - try it though